app/src/main/java/com/gogiyumandroid/RestaurantListActivity.kt

```diff
@@ -0,0 +1,247 @@
+package com.gogiyumandroid
+
+import android.annotation.SuppressLint
+import android.content.Intent
+import android.net.Uri
+import android.os.AsyncTask
+import android.os.Bundle
+import android.view.View
+import android.widget.*
+import android.widget.TableLayout
+import androidx.appcompat.app.AlertDialog
+import androidx.appcompat.app.AppCompatActivity
+import com.gogiyumandroid.RestaurantAdapter.RestaurantAdapter
+import kotlinx.android.synthetic.main.custom_restaurant_table_row.*
+import kotlinx.android.synthetic.main.custom_restaurant_table_row.view.*
+import kotlinx.android.synthetic.main.custom_table_row.view.*
+import okhttp3.*
+import org.json.JSONArray
+import java.net.URL
+
+
+@Suppress("DEPRECATION")
```

Comment:
What is this Suppress for? Is this class for deprecation?

Reply:
That part is not necessary anymore. so I erased it

Reply:
Nice job.